tests/payloads/PermissionedPayloadsController.t.sol

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IPayloadsControllerCore} from '../../src/contracts/payloads/interfaces/IPayloadsControllerCore.sol';
+import {IPermissionedPayloadsController, PermissionedPayloadsController} from '../../src/contracts/payloads/PermissionedPayloadsController.sol';
+import {PayloadsControllerUtils} from '../../src/contracts/payloads/PayloadsControllerUtils.sol';
+import {TransparentProxyFactory} from 'solidity-utils/contracts/transparent-proxy/TransparentProxyFactory.sol';
+import {Errors} from '../../src/contracts/libraries/Errors.sol';
+import {Executor, IExecutor, Ownable} from '../../src/contracts/payloads/Executor.sol';
+import {PayloadTest} from './utils/PayloadTest.sol';
+import {Test} from 'forge-std/Test.sol';
+
+contract PermissionedPayloadsControllerTest is Test {
+  address constant ADMIN = address(123);
```

Comment:
these addresses should be fuzzed

Reply:
It was copied from PayloadsControollerCore

Reply:
yup, there also we need to change, don't take to much stock on what is there, as when it was done fuzzing was not fully implemented yet. But if we do new tests, its better to always do fuzzing as much as we can